bitrepository-repository-settings-xsd/src/main/resources/xsd/RepositorySettings.xsd

```diff
@@ -212,13 +212,43 @@
           <xs:annotation>
             <xs:documentation>
               The timeout for the identification in milliseconds.
+              Deprecated for removal, but currently still mandatory for backward compatibility.
+            </xs:documentation>
+          </xs:annotation>
+        </xs:element>
+        <xs:element name="IdentificationTimeoutDuration" type="xs:duration" minOccurs="0" maxOccurs="1">
+          <xs:annotation>
+            <xs:documentation>
+              The timeout for the identification.
+              If present, clients should prefer to respect this setting over IdentificationTimeout.
+              Non-negative XML schema durations are accepted, for example
+              PT2H (2 hours), PT3M (3 minutes), PT1H22M22.5S (1 hour 22 minutes 22.5 seconds),
```

Comment:
I was unsure how much documentation we wanted. Should we expect people to know or lookup the syntax of an XML duration themselves, or is it fine to have a little guidance here? (The question will arise again in the reference project, where about 20 settings will be changed to use durations rather than milliseconds.)

Reply:
I think it's okay to expect people to know or lookup the duration themselves, but I think it's okay either way. Documentation is always nice imo. :)